src/components/Header/index.module.scss

```diff
@@ -16,6 +16,10 @@ $defaultHeaderHeight: 90px;
   }
 }
 
+.focusInvisible:focus-visible {
```

Comment:
The name is a bit weird

`focus invisible` => `focus visible`

What does it do

Reply:
> What does it do

There is a outline when button is clicked and has a [focus-visible](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) state.

<img width="173" alt="image" src="https://github.com/Magickbase/nervos-official-website/assets/3870972/94434a2d-baa3-425d-b24e-7659e4421f2e">

The purpose of a class with name of `focusInvisible` is to remove this outline by setting" 

```
outline: none;
```

for a HTML element with `focus-visible` state.

Changed the class name `focusInvisible` to `noFocusOutline` in https://github.com/Magickbase/nervos-official-website/pull/256/commits/d99313f0af24db315056429fc72a92939c5a136e